.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
   include:
     - env: NIX_PATH=nixpkgs=channel:nixpkgs-unstable
     - env: NIX_PATH=nixpkgs=channel:nixos-unstable
-    - env: NIX_PATH=nixpkgs=channel:nixos-20.03
+    - env: NIX_PATH=nixpkgs=channel:nixos-22.11
```

Comment:
The .travis.yml file has not been updated or modified for a long time (2 years), it seems that it is unused.

Should we remove it?

Reply:
I think so. Travis is pretty much dead.